plugins/modules/cloud/google/gce.py

```diff
@@ -21,68 +21,83 @@
     alternative: Use M(google.cloud.gcp_compute_instance) instead.
 options:
   image:
+    type: str
     description:
       - image string to use for the instance (default will follow latest
         stable debian image)
     default: "debian-8"
   image_family:
+    type: str
     description:
       - image family from which to select the image.  The most recent
         non-deprecated image in the family will be used.
   external_projects:
+    type: list
     description:
       - A list of other projects (accessible with the provisioning credentials)
         to be searched for the image.
   instance_names:
+    type: str
     description:
       - a comma-separated list of instance names to create or destroy
   machine_type:
+    type: str
     description:
       - machine type to use for the instance, use 'n1-standard-1' by default
     default: "n1-standard-1"
   metadata:
+    type: str
     description:
       - a hash/dictionary of custom data for the instance;
         '{"key":"value", ...}'
   service_account_email:
+    type: str
     description:
       - service account email
   service_account_permissions:
+    type: list
     description:
       - service account permissions (see
         U(https://cloud.google.com/sdk/gcloud/reference/compute/instances/create),
         --scopes section for detailed information)
-    choices: [
-      "bigquery", "cloud-platform", "compute-ro", "compute-rw",
-      "useraccounts-ro", "useraccounts-rw", "datastore", "logging-write",
-      "monitoring", "sql-admin", "storage-full", "storage-ro",
-      "storage-rw", "taskqueue", "userinfo-email"
-    ]
+      - >
+        Available choices are: [
```

Comment:
```suggestion
        Available choices are:
```
Same for the closing `]`.